compositions/README.md

```diff
@@ -38,3 +38,20 @@ Deploys VPC claim resource which uses the above composition.
 ```shell
 kubectl apply -f examples/terrajet-aws-provider/composition-resources/vpc.yaml
 ```
+
+# Configuration Packages
+
+Each folder contains a Crossplane configuration [package](https://crossplane.io/docs/v1.9/concepts/packages.html) definition which bundles all compositions into a single OCI image. 
+
+This is how you build and push the crossplane-aws-blueprints configuration package:
+```shell
+cd aws-provider
+export REPO=<some-oci-repository>
```

Comment:
I think it's better to set REPO to an example registry like `example-docker/crossplane-aws-blueprints` 

Then do 
```bash
kubectl crossplane push configuration $REPO:v0.0.1
```

Reply:
Done: https://github.com/aws-samples/crossplane-aws-blueprints/pull/51/commits/bde80f6b4f07ff0f747aa4e6441f5f3b675196e0